main.tf

```diff
@@ -0,0 +1,114 @@
+terraform {
+
+  backend "azurerm" {
+    resource_group_name  = "bearxapp_ProjectExercise"
+    storage_account_name = "storageaccounttfstate27"
+    container_name       = "containertfstate27"
+    key                  = "terraform.tfstate"
+  }
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 3.8"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+  
+  subscription_id = var.azure_subscription_id
+  client_id       = var.azure_client_id
+  client_secret   = var.azure_client_secret
+  tenant_id       = var.azure_tenant_id
+}
+
+data "azurerm_resource_group" "main" {
+    name = "bearxapp_ProjectExercise"
+
+}
+
+resource "random_string" "resource_code" {
```

Comment:
It looks like this is unused - probably pulled from a tutorial and then removed (correctly!) so it would be good to get rid of it now